test/sagemaker_tests/pytorch/training/conftest.py

```diff
@@ -509,11 +571,33 @@ def skip_pt20_cuda121_tests(
 
 
 @pytest.fixture(autouse=True)
-def skip_p5_tests(instance_type, processor, ecr_image):
-    if "p5." in instance_type:
-        image_cuda_version = get_cuda_version_from_tag(ecr_image)
-        if processor != "gpu" or Version(image_cuda_version.strip("cu")) < Version("120"):
-            pytest.skip("Images using less than CUDA 12.0 doesn't support P5 EC2 instance.")
+def skip_pt21_test(request):
+    if "framework_version" in request.fixturenames:
+        fw_ver = request.getfixturevalue("framework_version")
+    elif "ecr_image" in request.fixturenames:
+        fw_ver = request.getfixturevalue("ecr_image")
+    else:
+        return
+    if request.node.get_closest_marker("skip_pt21_test"):
+        if Version(fw_ver) in SpecifierSet("==2.1"):
```

Comment:
This is not going to work as expected for all versions of PT 2.1 - as 2.1.1 and 2.1.2, etc, will not resolve to True

Example:

```
In [3]: Version("2.1.2") in SpecifierSet("==2.1")
Out[3]: False
```

Update to 2.1.* if you want this to apply to all patch versions of 2.1